classes/production/grails-bootstrap/grails.build.properties

```diff
@@ -0,0 +1 @@
+grails.version=3.3.0.BUILD-SNAPSHOT
```

Comment:
It appears that syntax formatting **should** be obvious, given the UI buttons on the comment input box.